src/api/app/models/bs_request.rb

```diff
@@ -22,12 +22,18 @@ class SaveError < APIException
     setup 'request_save_error'
   end
 
-  scope :to_accept, -> { where(state: 'new').where('accept_at < ?', DateTime.now) }
+  scope :to_accept,
+   -> { where(state: 'new').where('accept_at < ?', DateTime.now) }
 
   has_many :bs_request_actions, -> { includes([:bs_request_action_accept_info]) }, dependent: :destroy
   has_many :reviews, :dependent => :delete_all
   has_and_belongs_to_many :bs_request_action_groups, join_table: :group_request_requests
-  has_many :comments, :dependent => :delete_all, inverse_of: :bs_request, class_name: 'CommentRequest'
+  has_many :comments, :dependent => :delete_all,
+        inverse_of:                   :bs_request,
+
+
+
+          class_name: 'CommentRequest'
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.